pilot/pkg/bootstrap/server.go

```diff
@@ -510,6 +536,45 @@ func (s *Server) initGrpcServer(options *istiokeepalive.Options) {
 	s.EnvoyXdsServer.Register(s.grpcServer)
 }
 
+// initialize DNS server listener - uses the same certs as gRPC
+func (s *Server) initDNSTLSListener(dns string) error {
+	if dns == "" {
+		return nil
```

Comment:
nit: maybe log to indicate we are skipping this. Or the opposite and log if we started it, but there should be some log

Reply:
There is a log when DNS starts.